amanuensis/config-default.yaml

```diff
@@ -215,6 +215,23 @@ AWS_SES:
 HUBSPOT:
   API_KEY: ""
   
-
-
-
+CONSORTIUM_STATUS:
+  INSTRUCT:
+    Code:
+    #The order matters.
+      - "REJECTED"
+      - "DATA_DELIVERED"
+      - "IN_REVIEW"
+      - "APPROVED"
+    Name:
+    #Order correspondes to codes.
+      - "Rejected"
+      - "Data Delivered"
+      - "In Review"
+      - "Approved"
```

Comment:
Could you refactor using a list of objects instead of two ordered list. It is less prone to human error when adding new states.
`CONSORTIUM_STATUS: 
  INSTRUCT: 
    - 
      Code: REJECTED
      Name: Rejected
    - 
      Code: DATA_DELIVERED
      Name: "Data Delivered"
    - 
      Code: IN_REVIEW
      Name: "In Review"
    - 
      Code: APPROVED
      Name: Approved`